app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,36 @@
+!!!
+%html
+  %head
+    %meta{:charset => "utf-8"}/
+    %title Instagram
+    %link{:href => "https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css", :rel => "stylesheet"}
+    %script{:src => "https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace<br>Line is too long. [87/80]